src/seedu/addressbook/AddressBook.java

```diff
@@ -381,7 +381,9 @@ public static String executeCommand(String userInputString) {
         final int maximumElements = 2;
         
 	final String[] split =  rawUserInput.trim().split("\\s+", maximumElements);
-        return split.length == maximumElements ? split : new String[] { split[0] , "" }; // else case: no parameters

```

Comment:
Did you do this using eclipse's auto formatting feature?  If not, check it out. This can be set in the auto-formatter. Show your teammates next time we meet.